keyboards/ash1800/info.json

```diff
@@ -0,0 +1,539 @@
+                    "x": 8.25,
+                    "y": 5
+                },
+                {
+                    "label": "<",
+                    "x": 9.25,
+                    "y": 5
+                },
+                {
+                    "label": ">",
+                    "x": 10.25,
+                    "y": 5
+                },
+                {
+                    "label": "?",
+                    "x": 11.25,
+                    "y": 5
+                },
+                {
+                    "label": "Shift",
+                    "x": 12.25,
+                    "y": 5,
+                    "w": 1.75
+                },
+                {
+                    "label": "1",
+                    "x": 15.5,
+                    "y": 5
+                },
+                {
+                    "label": "2",
+                    "x": 16.5,
+                    "y": 5
+                },
+                {
+                    "label": "3",
+                    "x": 17.5,
+                    "y": 5
+                },
+                {
+                    "label": "Enter",
+                    "x": 18.5,
+                    "y": 5,
+                    "h": 2
+                },
+                {
+                    "label": "up",
+                    "x": 14.25,
+                    "y": 5.25
+                },
+                {
+                    "label": "Ctrl",
+                    "x": 0,
+                    "y": 6,
+                    "w": 1.25
+                },
+                {
+                    "label": "OS",
+                    "x": 1.25,
+                    "y": 6,
+                    "w": 1.25
+                },
+                {
+                    "label": "Alt",
+                    "x": 2.5,
+                    "y": 6,
+                    "w": 1.25
+                },
+                {
+                    "x": 3.75,
+                    "y": 6,
+                    "w": 6.25
+                },
+                {
+                    "label": "Alt",
+                    "x": 10,
+                    "y": 6,
+                    "w": 1.5
+                },
+                {
+                    "label": "Ctrl",
+                    "x": 11.5,
+                    "y": 6,
+                    "w": 1.5
+                },
+                {
+                    "label": "0",
+                    "x": 16.5,
+                    "y": 6
+                },
+                {
+                    "label": ".",
+                    "x": 17.5,
+                    "y": 6
+                },
+                {
+                    "label": "left",
+                    "x": 13.25,
+                    "y": 6.25
+                },
+                {
+                    "label": "down",
+                    "x": 14.25,
+                    "y": 6.25
+                },
+                {
+                    "label": "right",
+                    "x": 15.25,
+                    "y": 6.25
```

Comment:
These lines should move up to between Lines 509 and 510. (Proper order should be Alt, Ctrl, Left, Down, Right, Numpad 0, Numpad Dot.)